Libplanet/Blockchain/BlockChain.cs

```diff
@@ -576,6 +577,7 @@ void WatchTip(object target, TipChangedEventArgs args)
                     () => Block<T>.Mine(
                         index: index,
                         difficulty: difficulty,
+                        previousTotalDifficulty: prevBlock.TotalDifficulty,
```

Comment:
Since prevBlock may not exist (which means this is the first block), the statement may cause error. I suggest:
```suggestion
                        previousTotalDifficulty: prevBlock?.TotalDifficulty ?? 0,
```